content/about/verslagen/17-18/2018-04-19.md

```diff
@@ -0,0 +1,174 @@
+    - **Penning (verplicht)**
+      De penningmeester houdt alle financiële zaken bij van de organisatie.
+      Deze functie is ook verplicht door de DSA, en kan misschien gecombineerd worden met secretaris, wegens het ook een administratieve taak is.
+    - **Secretaris**
+      De secretaris onderhoudt de administratie van het bestuur en schrijft de verslagen van de bestuursvergaderingen. Ook reserveert de secretaris de evenementen van Zeus en noteert ze in het DSA-controlepaneel. Deze taak kan door PR opgenomen worden.
+    - **Project**
+      De projectverantwoordelijke verzorgt de projecten in Zeus en zorgt voor een goede vooruitgang. Ook zorgt de projectverantwoordelijke voor de definitie van een goed development proces en onderhoudt de repos
+    - **Sysadmin**
+      De sysadmin verzorgt de servers, zij het lokaal, zij het in de cloud. Deze moeten altijd up to date gehouden worden zodat er geen security issues op kunnen komen.
+    - **PR**
+      De PR verantwoordelijke zorgt voor blogposts, communicatie met externen en onze social media. De PR zal dus ook foto's en dergelijke online zetten en het engagement met niet-Zeusleden verhogen
+    - **HR**
+      SHFEERBEHEER. Zorgt voor de algemene sfeer in de kelder en ledenwerking. Probeert actief leden aan te werven. Kennisoverdracht tussen events enzovoorts.
+      
+  Bestuursfuncties die we niet inrichten:
+    - **Vicevoorzitter (verplicht)**
+      Right hand (wo)man van de voorzitter. Wordt commander-in-chief indien de voorzitter niet beschikbaar is.
+      Deze functie is verplicht door DSA, maar in de realiteit is de vice gewoon secretaris met een fancy titel. In principe kan eender welk bestuurslid de vicevoorzitter zijn, ongeacht hun feitelijke rol.
+      Het voorstel is om deze rol intern aan te duiden onder de verkozen bestuursleden.
+    - **Event**
+      In plaats van één persoon constant te 'belasten' met het organiseren van alle events zouden we blijven bij de formule van dit jaar: nl. events in het begin van het semester te bespreken en een verantwoordelijke per event aan te duiden.
+    
+    Discussie
+  - HR samenvoegen met Vice + Vice beter omschrijven
+  - Projectverantwoordelijke is meer projecten overzien in tegenstelling tot coden.
+
+## 7. Trivia
+### Evaluatie vleor
+VLEOR (Dit is Zeusiaans voor: "De vloer ligt er en we zijn blij")
+
+### Blogpost ideeën/evaluatie
+
+We kunnen blogpost maken over:
+
+- bottlebats
+- update april
+- Datavisualisatie project Isaura, Elo en Lorin. Deze zouden visualisaties bevatten van Haldis, Tab, Tap, Zeus events, ...
+
+We zijn in ieder geval content over de recente stroom van blogposts.
+
+### Engagement bestuursleden.
+
+Is besproken geweest op de vergadering. Deze kon in sommige gevallen beter.
+
+### Opmerkingen Lorin over Linux Install Party volgend jaar
+
+Volgend jaar gaat dit huge zijn, dus meer organisatie (splitsen in 2 avonden bvb)
```

Comment:
Klopt dit wel? Er zijn altijd nauwelijks mensen die geen eerstejaars zijn op dat event. Gelijk begin dit jaar besproken komt dat volgens mij door de promo van bvb Dawyndt, dat het het eerste zeus-event is, en dat linux nog stoer, wild en onontgonnen lijkt wanneer je in je eerste week zit. Die dingen gelden allemaal niet voor mensen die een install party hebben gemist. Zelf zou ik gokken op een mild toegenomen opkomst.

Reply:
Tussen de eerstejaars van dit jaar zijn er vrij veel mensen die nog linux willen, en waarschijnlijk gaat dat volgend jaar ook nog wel tot een bepaald niveau zijn. We gaan dus misschien wel twee keer zoveel volk hebben.

Reply:
Hm, ik had verwacht dat die doelgroep hun handen al zelf vuil gemaakt zou hebben. I stand corrected!

Reply:
Discussiepuntje voor volgend jaar: Maken we dit het grootse Zeus-kickoff-event en zorgen we voor een zeer strakke organisatie waarin we zowel hulp bij het installeren als bij het gebruiken en tweaken voorzien? Als je dat goed uitvoert lijkt me dat een zeer goede en wenselijke eerste indruk voor eerstejaars, maar ook voor laterejaars die nog wat in UNIX te dabben hebben.

Reply:
Helemaal akkoord!
Ik zat zelf te denken om eventueel de LIP te doen in een pc-lokaal en zo wat practicum-achtig te maken.

Een Tiles & Terminals later in het eerste semester zou ook wel cool zijn.